.vscode/settings.json

```diff
@@ -2,17 +2,17 @@
     "[julia]": {
         "editor.formatOnSave": true
     },
-    "notebook.formatOnSave.enabled": false,
+    "notebook.formatOnSave.enabled": true,
     "notebook.codeActionsOnSave": {
-        // https://github.com/astral-sh/ruff-vscode/issues/593
-        "source.fixAll.ruff": "never",
-        "source.organizeImports.ruff": "never"
+        "notebook.source.fixAll": "explicit",
+        "notebook.source.organizeImports": "explicit"
```

Comment:
This was disabled due to the linked issue. Prefixing `notebook` as suggested there fixes the problem for me. The postfix ruff is removed, It is a way for [ruff-vscode](https://github.com/astral-sh/ruff-vscode) to avoid allowing other extensions to run on-save. That is not the default way to do it and I don't think there is an issue with that.